models/save.py

```diff
@@ -7,12 +7,16 @@ class Save():
     '''class to define a save'''
 
     __saves = 1
-    objects = []
+    objects = {
+            'save_1': {},
+            'save_2': {},
+            'save_3': {},
+            }
 
     def __init__(self):
         '''method to be called upon object instantiation'''
         self.save_id = self.__saves
-        self.__saves = self.save_id + 1
+        Save.__saves = self.save_id + 1
```

Comment:
if we're creating an object that all three saves are in, I'm not sure what the purpose of the auto-incrementing save id is anymore

Reply:
Ah yeah this was mistakenly left in good catch